frontend/components/tasks/affectiveAnnotation/inputs/Slider.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <v-container class="widget">
+    <v-row class="widget-row" justify="center" align="center">
+      <v-col :cols="(withCheckbox)?3:4" class="widget-row__category">
+        {{ categoryLabel }}
+      </v-col>
+      <v-col v-show="showSlider" :cols="(withCheckbox)?6:8" class="widget-row__slider">
+        <v-slider
+          :value="value"
+          min="0"
+          max="10"
+          step="1"
+          ticks
+          dense
+          :color="color"
+          track-color="grey"
+          :hint="hint"
+          persistent-hint
+          :thumb-size="sliderThumbSize"
+          :thumb-color="sliderThumbColor"
+          @click="markClicked"
+          @change="valueChangeHandler"
+        />
+      </v-col>
+      <v-col v-if="withCheckbox" cols="3" class="widget-row__checkbox">
+        <v-checkbox
+          :label="checkboxLabel"
+          @change="checkboxChangeHandler"
+        />
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script>
+import { mdiCheck } from '@mdi/js'
+
+export default {
+  props: {
+    categoryLabel: {
+      type: String,
+      default: ""
+    },
+    color: {
+      type: String,
+      default: "blue"
+    },
+    hint: {
+      type: String,
+      default: ""
+    },
+    value: {
+      type: Number,
+      default: 0
+    },
+    mustClick: {
```

Comment:
I wonder, is there any occurence in which the `mustClick` value can be false? Because I checked and I found no such occurence, but I might have missed something XD

Reply:
Thank you very much for the feedback! I still consider myself as a newbie with vue and nuxt stuff, so it means a lot to me to know I did it okay-ish. XD

> Additionally, I need to ask: is the behavior to reset the slider to 0 after clicking and reclicking nie wiem intentional?

Yes, it is intentional. It was related to the idea that when "nie wiem" is checked, then the value for the associated category should be set to -1 (like in our discussions before, we use -1 as a form of 'null' to distinguish it from true 0 and false). However, when the "nie wiem" checkbox is unchecked, the slider will go to -1 because the data was set to -1, but there is no -1 in the slider as the min value is 0, so it goes to 0.

I think it is possible to restore the previous value before the checkbox is checked, by putting it an additional variable, but it will add some more complexity and I just don't think it is worth the effort. XD

> I wonder, is there any occurence in which the mustClick value can be false? Because I checked and I found no such occurence, but I might have missed something XD

So far I think all requirements related to affective annotation make it obligatory to click on every slider (or either on the slider or the "nie wiem" checkbox). So, the answer is no, at least so far `mustClick` value in every usage of the slider component should always be set to true. XD

I made the `mustClick` prop just in case if in the future we want to make some of the sliders optional, in which case the slider will have the thumb appear right from the beginning with a default value of 0. 😅

Reply:
Alright, I understand XD 

Additionally, will this component be adjusted in the future? As for now, I think the development is very leaned towards Polish language, which is not bad if we were to compare it to the requirements, but it will not be consistent to the existing codebase   XD

Reply:
> Additionally, will this component be adjusted in the future? As for now, I think the development is very leaned towards Polish language, which is not bad if we were to compare it to the requirements, but it will not be consistent to the existing codebase XD

Yes, I will move all those polish words into the i18n directory and adjust the code accordingly. I think I will do that in the next PR while moving these components from article annotation page to affective annotation page XD

Reply:
or maybe I will do that in this PR, but I still need some more time 😅

Reply:
I think it will be better to do that in this PR, and the deadline is still far so I think it will be fine XD      I will be waiting for more updates then.